base/linalg/uniformscaling.jl

```diff
@@ -167,6 +167,8 @@ end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
 norm(J::UniformScaling, p::Real=2) = abs(J.λ)
+det(J::UniformScaling{T}) where {T} = one(T)
```

Comment:
Well, it is useful if you don't want to special case it. I opened this PR because I ran into that exact issue:

https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/problems/dde_problems.jl#L36